corehq/form_processor/backends/sql/dbaccessors.py

```diff
@@ -609,16 +611,22 @@ def soft_delete_forms(domain, form_ids, deletion_date=None, deletion_id=None):
 
         return affected_count
 
-    @staticmethod
     @transaction.atomic
-    def _archive_unarchive_form(form, user_id, archive):
+    def _archive_unarchive_form(self, form, user_id, archive, archive_stub):
+        form_id = form.form_id
+        with get_cursor(XFormInstanceSQL) as cursor:
+            cursor.execute('SELECT archive_unarchive_form(%s, %s, %s)', [form_id, user_id, archive])
```

Comment:
why did this need to get split out form the other SQL query?

Reply:
Because the logic in `_archive_unarchive_form` updates the history (but I can make the name of the function more clear) and `send_to_kafka` sends it to kafka (but maybe @calellowitz can double check that for me?). I was trying to do the same separation of logic here in sql that I put into the couch logic.

Reply:
I don't think it's necessary to separate these if we are marking the stub as 'updated' after these have been called.